### DISCUSSION TITLE
Error: permission denied for function crypto_aead_det_decrypt

### DISCUSSION THREAD

**Discussion by oliveirazicard** [supabase/supabase#27748] [Questions]
I have 2 columns with cryptography in my table "usuarios". 
I can insert data into the table in supabase client but I can not insert this via API from my front end application.

Who can help me about this problem, please?

![errosupabase](https://github.com/supabase/supabase/assets/29610750/366785ba-6184-403b-b5f0-8b79e11549af)

**Comment by GaryAustin1** [comment]
Maybe this:
https://github.com/supabase/supabase/issues/12952#issuecomment-1684263225

But note from the docs:
![image](https://github.com/supabase/supabase/assets/54564956/63cfc96b-a25f-4f0b-a3d2-ec83daff523a)
